tests/test_proxy.py

```diff
@@ -94,13 +93,6 @@ def test_recv_raises_ProxyCommandFailure_on_non_timeout_error(
         assert info.value.command == "hi"
         assert info.value.error == "whoops"
 
-    @patch("paramiko.proxy.subprocess.Popen")
-    @patch("paramiko.proxy.os.kill")
-    def test_close_kills_subprocess(self, os_kill, Popen):
```

Comment:
Is there another test that you could replace this test with, instead of only deleting it?